spec/calendar.html

```diff
@@ -290,10 +290,14 @@ <h1>DateFromFields ( _calendar_, _fields_, _options_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-temporal-yearmonthfromfields" aoid="YearMonthFromFields">
-      <h1>YearMonthFromFields ( _calendar_, _fields_, _options_ )</h1>
+      <h1>YearMonthFromFields ( _calendar_, _fields_ [ , _options_ ] )</h1>
       <emu-alg>
         1. Assert: Type(_calendar_) is Object.
         1. Assert: Type(_fields_) is Object.
+        1. If _options_ is not present, then
+          1. Set _options_ to *undefined*.
```

Comment:
I guess a conversation could be had about whether a normative change to make this into OrdinaryObjectCreate(**null**) is desired for consistency, since we do that in [one place](file:///var/home/ptomato/workspace/proposal-temporal/out/index.html#_ref_1327) in the spec text. But this change is indeed editorial and is correct.

I don't think the consistency thing is a big deal so I don't see a reason to change it unless an implementor brings it up.

Reply:
It'd only be normative if it ends up getting passed to other methods. Does it?

Reply:
Yes, it can potentially be passed to a `calendar.monthDayFromFields()` method in user code, so changing it to be something other than `undefined` would be observable.